packages/eslint-plugin-svelte/src/utils/svelte-context.ts

```diff
@@ -0,0 +1,259 @@
+import type { RuleContext } from '../types.js';
+import fs from 'fs';
+import path from 'path';
+import { getPackageJsons } from './get-package-json.js';
+import { getFilename, getSourceCode } from './compat.js';
+
+const isRunInBrowser = !fs.readFileSync;
+
+export type SvelteContext = (
+	| ({
+			svelteVersion: '3/4';
+	  } & {
+			svelteFileType: '.svelte' | null;
+			runes: null;
+	  })
+	| ({
+			svelteVersion: '5';
+	  } & (
+			| {
+					svelteFileType: '.svelte' | '.svelte.[js|ts]';
```

Comment:
```suggestion
					svelteFileType: '.svelte' | '.svelte.js';
```
In `svelteKitFileType`, TS extensions are omitted, I'd omit them here as well... (or add them there, either way, I'd do the same thing for both fields)

Reply:
changed it to js|ts.
https://github.com/sveltejs/eslint-plugin-svelte/pull/980/commits/4640ef535d92fdee4a404d1a74acedfaca22fcad

It states that it does not support coffeescript for now. (If someone raises an issue in the future, we may respond to it.)

Reply:
Ok, this works :)